### DISCUSSION TITLE
Flashing precompiled binary

### DISCUSSION THREAD

**Discussion by ScarsFun** [windowsair/wireless-esp8266-dap#32] [Q&A]
hi,
I'm trying to flash precompiled binary  to a nodemcu module.
I'm using expressif download tool. binary file :esp8266_dap_full.bin at address 0X0. It loads the binary file but, after reset, cannot discover it with wifi.
Sorry but I'm not an expert with Expressif tools. Just interested at DAP wifi debugger. 

Thank's in advance.

**Comment by windowsair** [comment]
hi, @ScarsFun 
I'm not sure if your firmware works. Please try to check the serial port output of esp8266.

**Reply by ScarsFun** [reply]
Starange chars from serial tryed at different speeds.
Aren't binaries from your releases uploadable to esp8266?

**Reply by windowsair** [reply]
> 

The firmware in release can of course be uploaded.

What I mean is that after you've upload it, re-power it and watch what's output from the serial port.  And there is some debugging information to help you determine what is going wrong with the wifi.

**Reply by windowsair** [reply]
Note that the baud rate should be `74880`.

**Reply by ScarsFun** [reply]
Strange chars... should I  set a particular speed?

**Reply by ScarsFun** [reply]
Ok ... read now . Il try as it possibile

**Reply by windowsair** [reply]
yep. `74880`

**Reply by ScarsFun** [reply]
debug response on console:
Disconnect reason : 201
Disconnect reason : 201
Disconnect reason : 201
Disconnect reason : 201
Disconnect reason : 201
Disconnect reason : 201
Disconnect reason : 201
Disconnect reason : 201

**Reply by windowsair** [reply]
this indicates that your AP was not found. is your AP set up correctly?

**Reply by ScarsFun** [reply]
found the problem: in windows 11, in mobile hotspot must specify 2.4Ghz band.
now I must set up usbip-win. no time now... later.

thanks for your help! I'll post my results here, may help someone else.

debug:

I (151914) wifi:state: 0 -> 2 (b0)
I (151917) wifi:state: 2 -> 3 (0)
I (151920) wifi:state: 3 -> 5 (10)
I (151929) wifi:connected with DAP, aid = 7, channel 11, HT20, bssid = da:f3:bc:73:06:f1
SYSTEM EVENT STA GOT IP : 192.168.137.116
Connected to AP
Socket created
Socket binded
Socket listening
Socket accepted
Handling dev attach request...
Sending header...
Sending device info...
* GET 0x02 CONFIGURATION DESCRIPTOR
Sending only first part of CONFIG
* GET 0x01 DEVICE DESCRIPTOR
* GET 0x02 CONFIGURATION DESCRIPTOR
Sending only first part of CONFIG
* GET 0x02 CONFIGURATION DESCRIPTOR
Sending ALL CONFIG
recv failed: errno 104
Shutting down socket and restarting...

**Reply by windowsair** [reply]
yes, esp32/8266 can not work at 5ghz.
